backend/kesaseteli/applications/models.py

```diff
@@ -16,6 +16,18 @@
 from companies.models import Company
 
 
+class School(HistoricalModel, TimeStampedModel, UUIDModel):
```

Comment:
Do we have any use-case for School Historical Model?

Reply:
Not really, probably inactive/active would be simpler. I'll look into this.

Reply:
Taken into account by removing HistoricalModel and adding deleted_at member variable and using it for filtering for active schools only.